tools/maxquant/macros.xml

```diff
@@ -96,6 +96,11 @@
             <requirement type="package" version="1.32">tar</requirement>
         </requirements>
     </xml>
+    <xml name="required_files">
+        <required_files>
+            <include path="mqparam.py" />
```

Comment:
@cat-bro I think we should add all file necessary. Maybe all python files?

Reply:
I've added create_mqpar.py (required by maxquant) and modify_mqpar.py (required by maxquant_mqpar).  I'm not sure if it is possible to say <include path="*.py"> here instead

Reply:
I think this should be fine.